examples/3d-tiles/app.js

```diff
@@ -18,14 +18,29 @@ const INDEX_FILE = `${DATA_URI}/modules/3d-tiles/test/data/index.json`;
 const MAPBOX_TOKEN = process.env.MapboxAccessToken; // eslint-disable-line
 const MAPBOX_STYLE = 'mapbox://styles/mapbox/light-v9';
 
+const ADDITIONAL_EXAMPLES = {
+  name: 'additional',
+  examples: {
+    royalExhibitionBuilding: {
+      tilesetUrl:
+        'https://raw.githubusercontent.com/uber-common/deck.gl-data/master/3d-tiles/RoyalExhibitionBuilding/tileset.json',
+      coordinateSystem: COORDINATE_SYSTEM.METER_OFFSETS,
```

Comment:
After this PR: We should try to create a transform that takes [0, 0, 0] to the cartesian that corresponds to the `coordinateOrigin` and update the tileset transform in the `deck.gl-data` repo.

that way we could treat this tileset the same as any other.

Reply:
Are you suggesting that Tileset3DLayer only takes data using WGS84 format?

Reply:
I don't have strong opinions on whether the `Tileset3DLayer` should be able to accept a coordinate system "override".

But I do think that the coordinate system origin and (orientation) should be stored in the tileset, not hard coded in the app.

As far as I can tell, the current 3D tile standard only supports WSG84 coordinates.

Reply:
PS Unless we plan to also export a `Tile3DLayer`, I think we should call the composite layer  `Tile3DLayer` instead of `Tileset3DLayer`. That should be simpler for users. It shows 3D tiles...